src/plugins/animation.js

```diff
@@ -15,7 +15,57 @@ export default properties => {
     errorSuggestions({ config: ['animation'] })
   }
 
+  const keyframes = {
+    spin: {
+      '@keyframes spin': {
+        from: {
+          transform: 'rotate(0deg)',
+        },
+        to: {
+          transform: 'rotate(360deg)',
+        },
+      },
+    },
+    ping: {
+      '@keyframes ping': {
+        '0%': {
+          transform: 'scale(1)',
+          opacity: 1,
+        },
+        '75%, 100%': {
+          transform: 'scale(2)',
+          opacity: 0,
+        },
+      },
+    },
+    pulse: {
+      '@keyframes pulse': {
+        '0%, 100%': {
+          opacity: 1,
+        },
+        '50%': {
+          opacity: 0.5,
+        },
+      },
+    },
+    bounce: {
+      '@keyframes bounce': {
+        '0%, 100%': {
+          transform: 'translateY(-25%)',
+          animationTimingFunction: 'cubic-bezier(0.8,0,1,1)',
+        },
+        '50%': {
+          transform: 'translateY(0)',
+          animationTimingFunction: 'cubic-bezier(0,0,0.2,1)',
+        },
+      },
+    },
+  }
+
+  const keyframe = keyframes[classValue] || {}
+
   return {
     animation: `${animationConfig}${important}`,
+    ...keyframe,
```

Comment:
I think I'll add a config value to disable adding the keyframes, just in case someone wants to use their own @keyframes for one of these animate classes within css or sass or something.

Reply:
Maybe `{ disableAutoKeyframes: true }`